tools/cli/commands/generate.js

```diff
@@ -418,12 +424,14 @@ async function mirrorRepo( composerJson, name, org = 'Automattic' ) {
 	} else if ( answers.newName ) {
 		await mirrorRepo( composerJson, answers.newName, org ); // Rerun this function so we can check if the new name exists or not, etc.
```

Comment:
You should add a `return` in here so line 434 doesn't overwrite what the recursive call did.

Reply:
Do you think it would be worth moving `composerJson.extra.autotagger = answers.autotagger` into the `addMirrorRepo` function instead and just passing it the monorepo questions?

Reply:
Sure, that could work too. I have no preference as to which way to do it.